apps/designer/app/designer/features/style-panel/controls/combobox-control.tsx

```diff
@@ -48,11 +48,13 @@ export const ComboboxControl = ({
 
   const setValue = setProperty(styleConfig.property);
 
+  // @todo abstract to a variant of the combobox component
+  // this whole switch block is a stop-gap measure, because i worked on flex-controls before the combobox update
+  // the combobox should be updated to supporting this rendered production
```

Comment:
Is ds combobox is missing something that is required for this?

Reply:
it should allow for any children, this should be easy to change right away, no?

Reply:
looking at it now its weird that we made items render strings instead of any jsx